src/System.Net.NetworkInformation/pkg/System.Net.NetworkInformation.pkgproj

```diff
@@ -2,9 +2,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\4.0.0\System.Net.NetworkInformation.depproj">
```

Comment:
I believe I already made this change in master: https://github.com/dotnet/corefx/commit/d5ea64de4e7092aed686faabfa2a638fe7a322b8#diff-5ae352d2ccfcfc4109f346d6da5902b9L5